### DISCUSSION TITLE
Better(better use and ux) support for doing api gateway

### DISCUSSION THREAD

**Discussion by bigpod98** [dotnet/yarp#1834] [Ideas] [answered]
While you can already use yarp as apu gateway its quite complex to get it to that level it would be awsome to have it be easier to do.

**Comment by MihaZupan** [comment, marked as answer]
Hey, can you share any specifics about what you were trying to achieve and what you had trouble with (available  docs / samples / APIs)?

**Reply by bigpod98** [reply]
Sorry no samples right now but One of the things i failed at and wanted to do was shaping of sorts, take a post request and change it into get request with things in vody being a part of url, or other way around.

**Reply by adityamandaleeka** [reply]
Triage: You should be able to do this today using transforms (if you're careful). However, this type of thing isn't a focus for our project so we don't plan to invest in making it smoother.
